packages/radix-ui-themes/src/props/prop-def.ts

```diff
@@ -73,4 +79,13 @@ type GetPropDefTypes<P> = {
   [K in keyof P]?: GetPropDefType<P[K]>;
 };
 
-export type { PropDef, ResponsivePropDef, GetPropDefTypes };
+export { breakpoints };
+export type {
+  PropDef,
+  GetPropDefTypes,
```

Comment:
I kind of feel like `GetPropDefTypes` is more of a helper, but didn't want to put it into helpers because we'd end up with helpers importing props and props importing helpers